crates/router/src/routes/payments.rs

```diff
@@ -695,8 +695,61 @@ pub async fn payments_connector_session(
     state: web::Data<app::AppState>,
     req: actix_web::HttpRequest,
     json_payload: web::Json<payment_types::PaymentsSessionRequest>,
```

Comment:
why do we need a payload in this request? since this is a get request

Reply:
this is post request.